test-data/unit/check-class-namedtuple.test

```diff
@@ -482,3 +483,76 @@ Y(y=1, x='1').method()
 class CallsBaseInit(X):
     def __init__(self, x: str) -> None:
         super().__init__(x)
+
+[case testNewNamedTupleWithMethods]
+# flags: --python-version 3.6
```

Comment:
I don't feel strongly about it, but it seems mildly useful to call out that this code will only work in 3.6 and not earlier versions.